compact/ecal/barrel_sciglass.xml

```diff
@@ -44,6 +44,11 @@
                   zmax="EcalBarrelForward_zmax" zmin="-EcalBarrelBackward_zmax"
                   etamin="EcalBarrel_etamin" etamax="EcalBarrel_etamax"/>
 
+      <documentation>
+        Pick the material for the towers
+      </documentation>
+      <material name="SciGlass" />
```

Comment:
A more typical approach is to add `material` as an attribute to the structure that defines the object that should receive that material. Without that, this sort of stands on its own without context so it may be unclear what the material exactly refers to.

Reply:
Yeah, thought about this. The other option is to put the material as an attribute into each family, or add a `<towers />` tag.